Makefile.pseudomodules

```diff
@@ -2,3 +2,4 @@ PSEUDOMODULES += defaulttransceiver
 PSEUDOMODULES += transport_layer
 PSEUDOMODULES += pktqueue
 PSEUDOMODULES += ng_netbase
+PSEUDOMODULES += newlib
```

Comment:
Because ```syscalls.c``` has to be in front of the linker command line, and the newlib module's Makefile.include adds it there (through ```UNDEF```). So in order not to add newlib.a again, I've added it to PSEUDOMODULES.